3.2/s2i/bin/usage

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+man -l /help.1
```

Comment:
The `/help.1` is meant to be formatted for `atomic` tool that shows the help file using `groff` tool. Using just pure `groff` would also limit dependencies, since `man` requires more deps. So, `man -l /help.1` may be replaced by `groff -man -ETascii /help.1`.